CHANGELOG.md

```diff
@@ -1,11 +1,18 @@
 # Changelog
 ---
 ### V2.1:
-        1.Seperated connection and messaging into a different .h file called Connection.h
-        2.Seperated pin setup into PinLogic.h file.
-        3.Home.ino added two includes of Connection.h & PinLogic.h.
-        4.Home.ino now calls function from .h libraries.
-        5.Added new Original Source Code V.1-2 folder.
-            i.  Added original source code SourceCodeV1 for reviewing purposes.
-            ii. Added V2 of source code as SourceCodeV2 for reviewing purposes.
-        6.Moved V2.1 code all into a folder named V2.1.
+        1.Added information to ReadMe document about Home.ino
+            i. Basic description of code. 
+        2.Added information to ReadMe document about PinLogic.h 
+            i. Basic description of code.
+        3.Added comment lines for Home.ino
+        4.Added comment lines for PinLogic.h 
+---
+Removed from PinLogic.h
+
+Reason: Not needed, code was not removed before V2.1. 
+```c++
+        int spin = digitalRead(pin); 
+        int butval = 0;       
```

Comment:
Space this in a little like this. I use 3 spaces from the beginning and then follow the code indentation from there when more lines are added to a block like this.
```c++
   int spin = digitalRead(pin); 
   int butval = 0;
```